src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  transition: $transition;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__information {
+    margin-top: 40px;
+  }
+
+  &__title {
```

Comment:
I didn't see your hover for title. Checklist: change the card title text color to `#34568b` when the card is hovered (`.card:hover .card__title`)